### PR TITLE
Add Chinese Translation of deployment.md.

### PR DESCRIPTION
**Pull request by fatfishZhao**
The translation is from the latest english version.

### CODE REVIEW

**Review by gaotongxiao — COMMENTED** [review]
Hi, thanks for your translation! It hits our guidelines except for the missing spaces between English and Chinese characters. Could you go over it again and fix it?

**Comment by gaotongxiao** [comment]
Also don't forget to sign CLA

**Comment by fatfishZhao** [comment]
> [CLA assistant check](https://cla-assistant.io/open-mmlab/mmocr?pullRequest=506) 
> Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/open-mmlab/mmocr?pullRequest=506) before we can accept your contribution.
> 
> **zhaoyuxuan** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).
> You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/open-mmlab/mmocr?pullRequest=506) it.
Hi. I have already signed the CLA. That username is the username on my Mac and I did not set my email when committing. How can I fix this problem? Do I need to set the local email on Mac and re-commit?

**Comment by gaotongxiao** [comment]
Yes, you can use `git commit --amend` and force push your branch after setting up your email address locally.

File: docs_zh_CN/deployment.md [hunk 1]
```diff
@@ -21,44 +21,52 @@ python tools/deployment/pytorch2onnx.py
...
+| `--output-file`    | str            | 输出的 ONNX 模型路径。 默认为 `tmp.onnx`。                   |
+| `--device-id`      | int            | 使用哪块 GPU。默认为0。                                      |
+| `--opset-version`  | int            | ONNX 操作集版本。默认为11。                                  |
+| `--verify`         | bool           | 决定是否验证输出模型的正确性。默认为 `False`。               |
+| `--verbose`        | bool           | 决定是否打印导出模型的结构，默认为 `False`。                 |
+| `--show`           | bool           | 决定是否可视化 ONNXRuntime 和 PyTorch 的输出。默认为 `False`。 |
+| `--dynamic-export` | bool           | 决定是否导出有动态输入和输出尺寸的 ONNX 模型。默认为 `False`。 |
 
-**Note**: This tool is still experimental. Some customized operators are not supported for now. And we only support `detection` and `recognition` for now.
+:::{注}
```
Comment (gaotongxiao) [inline docs_zh_CN/deployment.md:40]:
```suggestion
:::{note}
```
It is just a directive for our docs parser, so we don't need to translate it here.

File: docs_zh_CN/deployment.md [hunk 2]
```diff
@@ -21,44 +21,52 @@ python tools/deployment/pytorch2onnx.py
...
 | PSENet |    [psenet_r50_fpnf_600e_icdar2015.py](https://github.com/open-mmlab/mmocr/blob/main/configs/textdet/psenet/psenet_r50_fpnf_600e_icdar2015.py)   |       Y       |        Y        |      |
 |  PANet |   [panet_r18_fpem_ffm_600e_ctw1500.py](https://github.com/open-mmlab/mmocr/blob/main/configs/textdet/panet/panet_r18_fpem_ffm_600e_ctw1500.py)   |       Y       |        Y        |      |
 |  PANet | [panet_r18_fpem_ffm_600e_icdar2015.py](https://github.com/open-mmlab/mmocr/blob/main/configs/textdet/panet/panet_r18_fpem_ffm_600e_icdar2015.py) |       Y       |        Y        |      |
-|  CRNN  |             [crnn_academic_dataset.py](https://github.com/open-mmlab/mmocr/blob/main/configs/textrecog/crnn/crnn_academic_dataset.py)            |       Y       |        Y        | CRNN only accepts input with height 32 |
+|  CRNN  |             [crnn_academic_dataset.py](https://github.com/open-mmlab/mmocr/blob/main/configs/textrecog/crnn/crnn_academic_dataset.py)            |       Y       |        Y        | CRNN 仅接受高度为32的输入 |
+
+:::{注}
```
Comment (gaotongxiao) [inline docs_zh_CN/deployment.md:59]:
```suggestion
:::{note}
```

File: docs_zh_CN/deployment.md [hunk 3]
```diff
@@ -77,51 +85,59 @@ python tools/deployment/onnx2tensorrt.py
...
+| `--trt-file`       | str            | 输出的 TensorRT 模型路径。默认为 `tmp.trt`。                |
+| `--max-shape`      | int * 4        | 模型输入的最大尺寸。                                        |
+| `--min-shape`      | int * 4        | 模型输入的最小尺寸。                                        |
+| `--workspace-size` | int            | 最大工作空间大小，单位为 GiB。默认为1。                     |
+| `--fp16`           | bool           | 决定是否输出 fp16 模式的 TensorRT 模型。默认为 `False`.     |
+| `--verify`         | bool           | 决定是否验证输出模型的正确性。默认为 `False`。              |
+| `--show`           | bool           | 决定是否可视化 ONNX 和 TensorRT 的输出。默认为 `False`。    |
+| `--verbose`        | bool           | 决定是否在创建 TensorRT 引擎时打印日志信息。默认为 `False`. |
+
+:::{注}
```
Comment (gaotongxiao) [inline docs_zh_CN/deployment.md:105]:
```suggestion
:::{note}
```

File: docs_zh_CN/deployment.md [hunk 4]
```diff
@@ -77,51 +85,59 @@ python tools/deployment/onnx2tensorrt.py
...
 | PSENet |    [psenet_r50_fpnf_600e_icdar2015.py](https://github.com/open-mmlab/mmocr/blob/main/configs/textdet/psenet/psenet_r50_fpnf_600e_icdar2015.py)   |       Y       |        Y        |      |
 |  PANet |   [panet_r18_fpem_ffm_600e_ctw1500.py](https://github.com/open-mmlab/mmocr/blob/main/configs/textdet/panet/panet_r18_fpem_ffm_600e_ctw1500.py)   |       Y       |        Y        |      |
 |  PANet | [panet_r18_fpem_ffm_600e_icdar2015.py](https://github.com/open-mmlab/mmocr/blob/main/configs/textdet/panet/panet_r18_fpem_ffm_600e_icdar2015.py) |       Y       |        Y        |      |
-|  CRNN  |             [crnn_academic_dataset.py](https://github.com/open-mmlab/mmocr/blob/main/configs/textrecog/crnn/crnn_academic_dataset.py)            |       Y       |        Y        | CRNN only accepts input with height 32 |
+|  CRNN  |             [crnn_academic_dataset.py](https://github.com/open-mmlab/mmocr/blob/main/configs/textrecog/crnn/crnn_academic_dataset.py)            |       Y       |        Y        | CRNN 仅接受高度为32的输入 |
 
-**Notes**:
+::{注}
```
Comment (gaotongxiao) [inline docs_zh_CN/deployment.md:124]:
```suggestion
:::{note}
```

File: docs_zh_CN/deployment.md [hunk 5]
```diff
@@ -292,8 +309,11 @@ python tools/deploy_test.py \
...
-- Note that, changing upsampling mode reduces less performance compared with using nearst mode. However, the weights of networks are trained through nearst mode. To persue best performance, using bilinear mode for both training and TensorRT deployment is recommanded.
-- All ONNX and TensorRT models are evaluated with dynamic shape on the datasets and images are preprocessed according to the original config file.
-- This tool is still experimental, and we only support `detection` and `recognition` for now.
+:::{注}
+
```
Comment (gaotongxiao) [inline docs_zh_CN/deployment.md:313]:
```suggestion
:::{note}
```

File: docs_zh_CN/deployment.md [hunk 6]
```diff
@@ -292,8 +309,11 @@ python tools/deploy_test.py \
...
+- TensorRT 上采样（upsample）操作和 PyTorch 有一点不同。对于 DBNet 和 PANet，我们建议把上采样的最近邻 (nearest) 模式代替成双线性 (bilinear) 模式。 PANet 的替换处在[这里](https://github.com/open-mmlab/mmocr/blob/50a25e718a028c8b9d96f497e241767dbe9617d1/mmocr/models/textdet/necks/fpem_ffm.py#L33) ，DBNet 的替换处在[这里](https://github.com/open-mmlab/mmocr/blob/50a25e718a028c8b9d96f497e241767dbe9617d1/mmocr/models/textdet/necks/fpn_cat.py#L111) 和 [这里](https://github.com/open-mmlab/mmocr/blob/50a25e718a028c8b9d96f497e241767dbe9617d1/mmocr/models/textdet/necks/fpn_cat.py#L121) 。如在上表中显示的，带有标记*的网络的上采样模式均被改变了。
+- 注意到，相比最近邻模式，使用更改后的上采样模式会降低性能。然而，默认网络的权重是通过最近邻（nearest）模式训练的。为了保持在部署中的最佳性能，建议在训练和 TensorRT 部署中使用双线性（bilinear）模式。
+- 所有 ONNX 和 TensorRT 模型都使用数据集上的动态尺寸进行评估，图像根据原始配置文件进行预处理。
+- 这个工具仍然是试验性的。一些定制的操作模型没有被支持。并且我们目前仅支持一部分的文本检测和文本识别算法。
+
```
Comment (gaotongxiao) [inline docs_zh_CN/deployment.md:318]:
```suggestion
```

File: docs_zh_CN/deployment.md [hunk 7]
```diff
@@ -77,51 +85,59 @@ python tools/deployment/onnx2tensorrt.py
...
-We provide methods to evaluate TensorRT and ONNX models in `tools/deployment/deploy_test.py`.
+### 评估 ONNX 和 TensorRT 模型（试验性的）
 
-#### Prerequisite
-To evaluate ONNX and TensorRT models, onnx, onnxruntime and TensorRT should be installed first. Install `mmcv-full` with ONNXRuntime custom ops and TensorRT plugins follow [ONNXRuntime in mmcv](https://mmcv.readthedocs.io/en/latest/onnxruntime_op.html) and [TensorRT plugin in mmcv](https://github.com/open-mmlab/mmcv/blob/master/docs/tensorrt_plugin.md).
+我们在 `tools/deployment/deploy_test.py ` 中提供了评估 TensorRT 和 ONNX 模型的方法。
 
-#### Usage
+#### 前提条件
+在评估 ONNX 和 TensorRT 模型之前，首先需要安装 ONNX，ONNXRuntime 和 TensorRT。根据[ONNXRuntime in mmcv](https://mmcv.readthedocs.io/en/latest/onnxruntime_op.html) and [TensorRT plugin in mmcv](https://github.com/open-mmlab/mmcv/blob/master/docs/tensorrt_plugin.md)安装 ONNXRuntime 定制操作和 TensorRT 插件。
```
Comment (gaotongxiao) [inline docs_zh_CN/deployment.md:138]:
```suggestion
在评估 ONNX 和 TensorRT 模型之前，首先需要安装 ONNX，ONNXRuntime 和 TensorRT。根据 [ONNXRuntime in mmcv](https://mmcv.readthedocs.io/en/latest/onnxruntime_op.html) and [TensorRT plugin in mmcv](https://github.com/open-mmlab/mmcv/blob/master/docs/tensorrt_plugin.md) 安装 ONNXRuntime 定制操作和 TensorRT 插件。
```

**Review by gaotongxiao — COMMENTED** [review]
Thanks! Here are some final editing suggestions.

**Comment by fatfishZhao** [comment]
OK. These notes are fixed.

**Comment by fatfishZhao** [comment]
Deleted trailing-whitespace.

**Review by gaotongxiao — APPROVED** [review]

**Pull request merged**